NEWS.md

```diff
@@ -1,6 +1,11 @@
 
 # vctrs 0.2.99.9000
 
+* New experimental API for tabular vectors. It currently consists in
```

Comment:
Is "tabular vector" the right name for this? I'm slightly worried that `tbl` is going to be easily confused with tibbles.